nujo/utils/data/dataset.py

```diff
@@ -13,31 +13,35 @@ class Dataset:
 
     Parameters:
     -----------
-    name : will be downloaded from the UCI ML repo (for now)
+    - name : str, will be downloaded from the UCI ML repo (for now)
+    - type : str, type of dataset - either csv or image
+    - override : boolean, for loader, will override file if it exists
+    - download : boolean, should it be downloaded from uci repo
 
-    Returns:
-    --------
-    array : stores the csv dataset,
-    - floating point integers
-    - last column -> labels
 
     '''
-    def __init__(self, name: str):
-        self.name = HOME_DIR + name
-        if '.data' not in name:
-            self.name += '.data'
-        DatasetLoader(self)
+    def __init__(self,
+                 name: str,
+                 type: str,
+                 override: bool = True,
+                 download: bool = False):
```

Comment:
Don't type hint here since it's obvious what the type is, you assign it to bool literal.
Also, add a new line after the method signature, when the signature needs more than one line of code.